### DISCUSSION TITLE
Updating existing QSOs with QRZ info

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#810] [Ideas / Feature Requests]
Is there a mechanism whereby someone could update an existing QSO with more info from a QRZ profile for a callsign? For example, the gridsquare?

**Comment by magicbug** [comment]
Currently not, but on the roadmap

**Reply by magicbug** [reply]
You can see the roadmap at https://github.com/magicbug/Cloudlog/projects/8 currently putting together the plan for 2021
